lib/cpp/src/thrift/concurrency/TimerManager.cpp

```diff
@@ -52,6 +52,8 @@ class TimerManager::Task : public Runnable {
     }
   }
 
+  shared_ptr<Runnable> runnable() const { return runnable_; }
```

Comment:
Consider implementing an ``operator ==`` instead which compares to a ``shared_ptr<Runnable>`` thus hiding the internal details of the ``TimerManager::Task`` and simplifying the caller to ``if (task == ix->second)`` below.  What's there works, just a suggestion - not required to merge.

Reply:
done